postinstall.sh

```diff
@@ -6,9 +6,12 @@ set -x
 # If there is a libffmpegsumo, copy it over
 if test -f /opt/google/chrome/libffmpegsumo.so; then
 	cp /opt/google/chrome/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
+elif test -f /usr/lib/chromium/libffmpegsumo.so; then
+	cp /usr/lib/chromium/libffmpegsumo.so node_modules/nw/nwjs/libffmpegsumo.so
 else
 	echo "Couldn't locate libffmpegsumo.so during installation. Audio notifications will be disabled" 1>&2
-	echo "To repair this, please install Google Chrome and reinstall \`plaidchat\`" 1>&2
+	echo "To repair this, please install Google Chrome/Chromium and reinstall \`plaidchat\`" 1>&2
+	echo "Else you can copy the libffmpegsumo.so file in node_modules/nw/nwjs/" 1>&2
```

Comment:
Right, I know that you can find it. But I am saying that most users will be confused and have no context.

If we want to solve this problem in a more reusable manner (e.g. will still work when we move to a `.deb`/`.rpm` flavor), then maybe we could build a `plaidchat` command that copies over a `.so` file? But I think that would belong in another PR.

```bash
plaidchat install-audio path/to/libffmpegsumo.so
```